src/js/services/notificationsService.js

```diff
@@ -57,23 +57,23 @@ angular.module('copayApp.services')
 
       switch (notificationData.type) {
         case 'NewTxProposal':
-          notification.new(gettext('New Transaction'),
+          notification.new(gettext('New Payment Proposal'),
             name, {color: color} );
           break;
         case 'TxProposalAcceptedBy':
-          notification.success(gettext('Transaction Signed'),
+          notification.success(gettext('Payment Proposal Signed by Copayer'),
             name, {color: color} );
           break;
         case 'TxProposalRejectedBy':
-          notification.error(gettext('Transaction Rejected'),
+          notification.error(gettext('Payment Proposal Rejected by Copayer'),
             name, {color: color} );
           break;
         case 'TxProposalFinallyRejected':
-          notification.error(gettext('A transaction was finally rejected'),
+          notification.error(gettext('Payment Proposal Finally Rejected'),
```

Comment:
What is this particular message used for?  "Finally" has a sarcastic tone to it, maybe it should be something like: "Payment Proposal was Rejected and Closed"